### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by angao**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/docs/imported/release/notes.md [hunk 1]
```diff
@@ -817,7 +817,7 @@ And point to the file using the --config flag, as in
...
 
 * `kubectl port-forward` now supports specifying a service to port forward to, as in `kubectl port-forward svc/myservice 8443:443`. Additional support has also been added for looking up targetPort for a service, as well as enabling using svc/name to select a pod. ([#59809](https://github.com/kubernetes/kubernetes/pull/59809), [@phsiao](https://github.com/phsiao))
-* [Make NodePort IP addres](https://github.com/kubernetes/website/pull/7631/files)[ses configurabl](https://github.com/kubernetes/website/pull/7631/files)[e](https://github.com/kubernetes/website/pull/7631/files). ([#58052](https://github.com/kubernetes/kubernetes/pull/58052), [@m1093782566](https://github.com/m1093782566))
+* [Make NodePort IP address](https://github.com/kubernetes/website/pull/7631/files)[ses configurabl](https://github.com/kubernetes/website/pull/7631/files)[e](https://github.com/kubernetes/website/pull/7631/files). ([#58052](https://github.com/kubernetes/kubernetes/pull/58052), [@m1093782566](https://github.com/m1093782566))
```
Comment (tengqm) [inline content/en/docs/imported/release/notes.md]: Read the whole sentence and then try revise it again?

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ content_template: templates/user-journey-content
...
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculous mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculous mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Don't do this ... unless you know what you are doing.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 3]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Same here.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 4]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Again.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 5]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Here too.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 6]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: It is a waste of life.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 7]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Spend your time on something meaningful.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 8]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: This is not fun.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 9]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Please.

File: content/en/docs/user-journeys/users/cluster-operator/_advanced.md [hunk 10]
```diff
@@ -26,55 +26,55 @@ Network-level service management (ISTIO)
...
```
Comment (tengqm) [inline content/en/docs/user-journeys/users/cluster-operator/_advanced.md]: Stop doing this.

File: content/en/docs/imported/release/notes.md (hunk 1 above)
Comment (angao) [reply content/en/docs/imported/release/notes.md]: ok, but that's a typo

**Comment by steveperry-53** [comment]
@angao Please remove the _advanced file from this PR.

**Pull request closed without merging**